lib/async.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const fs = require('fs');
+const util = require('util');
+const faker = require('faker');
+let path = process.argv.slice(2)[0];
+let file = process.cwd()+`/data/${path}`;
+
+
+const promise = util.promisify(fs.readFile);
+const promisewrite = util.promisify(fs.writeFile);
+
+
+async function readAsync(file){
+  await promise(file)
+    .then(data => {
+      data = JSON.parse(data);
+      console.log('async before', data);
+      data.firstName = faker.name.firstName();
+      data = JSON.stringify(data);
+      promisewrite(file, data);
+      console.log('async after', data);
+    })
+    .catch(error => error);
+}
+readAsync(file);
```

Comment:
Ah, i see now why your `index.js` was blank. This is an improper way to use module files; we don't want our `require` statement to cause a function execute. Instead, we want our modules to define functions and then export those functions without running them. Then when we `require` that module, we have control over when we call that function.